### PR TITLE
Multichain Governance Docs

### PR DESCRIPTION
**Pull request by ajb413**

### CODE REVIEW

File: docs/pages/v2/governance.md [hunk 1]
```diff
@@ -592,7 +592,7 @@ const tx = await gov.methods.castVoteBySig(proposalId, 1, v, r, s).send({});
 
 ## Timelock
 
-Each protocol contract is controlled by the Timelock contract, which can modify system parameters, logic, and contracts in a 'time-delayed, opt-out' upgrade pattern. The Timelock has a hard-coded minimum delay of 2 days, which is the least amount of notice possible for a governance action.
+Each protocol contract is controlled by the [Timelock contract](https://github.com/compound-finance/compound-protocol/blob/master/contracts/Timelock.sol){:target="_blank"}, which can modify system parameters, logic, and contracts in a 'time-delayed, opt-out' upgrade pattern. The Timelock has a hard-coded minimum delay of 2 days, which is the least amount of notice possible for a governance action.
```
Comment (scott-silver) [inline docs/pages/v2/governance.md:595]: > The Timelock has a hard-coded minimum delay of 2 days, which is the least amount of notice possible for a governance action.

That is true for the mainnet governance timelock, but just as an FYI, the L2 Timelocks can have their minimum delay set to other values. For example, the minimum delay for the Polygon timelock is only 12 hours

File: docs/pages/v3/governance.md [hunk 2]
```diff
@@ -56,6 +56,14 @@ Each time an immutable parameter is set via governance proposal, a new Comet imp
 
 To set specific protocol parameters in a proposal, the Timelock must call all of the relevant set methods on the *Configurator* contract, followed by `deployAndUpgradeTo` on the *CometProxyAdmin* contract.
 
+## Governance on Other EVM Chains
+
+The Compound III protocol can be deployed on any EVM chain. The deployment must have access to on-chain asset prices and messages passed from Ethereum Mainnet. The [Timelock](/v2/governance/#timelock) on Mainnet is the administrator of all community sanctioned instances of Compound III.
+
+Each deployment outside of Mainnet needs to have a [Bridge Receiver](https://github.com/compound-finance/comet/blob/main/contracts/bridges/BaseBridgeReceiver.sol) and Local Timelock contract on its chain. Governance proposals executed on Ethereum must be read by the chain's bridge and published to the Bridge Receiver. Local Timelocks have an additional delay of one day before Comet admin functions can be called.
```
Comment (scott-silver) [inline docs/pages/v3/governance.md:63]: > Local Timelocks have an additional delay of one day

The delay is configurable, so it's not necessarily one day. It's whatever value was set at contract creation

**Comment by scott-silver** [comment]
Nice!

**Review by kevincheng96 — APPROVED** [review]
Echoing Scott's comments. Otherwise LGTM!

**Review by scott-silver — APPROVED** [review]

File: docs/pages/v2/governance.md (hunk 1 above)
Comment (ajb413) [reply docs/pages/v2/governance.md:595]: Good call, this isn't even true on mainnet anymore, I think it is one day there.

**Pull request merged**